idb/postgres/internal/encoding/encode.go

```diff
@@ -2,10 +2,11 @@ package encoding
 
 import (
 	"encoding/base64"
-
+	sdk_types "github.com/algorand/go-algorand-sdk/types"
```

Comment:
I think there should be an empty line to separate standard libraries from non-standard imports.